### PR TITLE
Added welsh translation for joint app when app2 request changes

### PR DESCRIPTION
**Pull request by nitinprabhu**
### Change description ###

Notification template https://www.notifications.service.gov.uk/services/56f68e5c-b51f-4bdf-a0ac-d70af335e1d1/templates/92191d4c-6078-4859-ad4d-8320d49b262d

### JIRA link (if applicable) ###

https://tools.hmcts.net/jira/browse/NFDIV-2020

**Before merging a pull request make sure that:**

- [x] tests have been updated / new tests has been added (if needed)
- [x] README and other documentation has been updated / added (if needed)

**If this ticket will have any visible impact on users and is not behind a feature toggle, make sure that:**
- [ ] this ticket been reviewed by QA
- [ ] the user story been signed off by the PO

### CODE REVIEW

File: src/main/java/uk/gov/hmcts/divorce/citizen/notification/Applicant2RequestChangesNotification.java [hunk 1]
```diff
@@ -88,11 +89,18 @@ public void sendToApplicant2(final CaseData caseData, final Long id) {
             );
         } else {
             log.info("Sending notification to applicant 2 to confirm their request for changes (applicant 1 is not represented): {}", id);
+            final var templateContent = commonContent.mainTemplateVars(caseData, id, caseData.getApplicant2(), caseData.getApplicant1());
+
+            LanguagePreference languagePreference = caseData.getApplicant2().getLanguagePreference();
+
+            if (LanguagePreference.WELSH.equals(languagePreference)) {
+                templateContent.put(CommonContent.PARTNER, commonContent.getPartnerWelshContent(caseData, caseData.getApplicant2()));
+            }
```
Comment (mapingo) [inline src/main/java/uk/gov/hmcts/divorce/citizen/notification/Applicant2RequestChangesNotification.java:98]: Shouldn't the partner be applicant 1

File: src/main/java/uk/gov/hmcts/divorce/citizen/notification/Applicant2RequestChangesNotification.java [hunk 2]
```diff
@@ -88,11 +89,18 @@ public void sendToApplicant2(final CaseData caseData, final Long id) {
             );
         } else {
             log.info("Sending notification to applicant 2 to confirm their request for changes (applicant 1 is not represented): {}", id);
+            final var templateContent = commonContent.mainTemplateVars(caseData, id, caseData.getApplicant2(), caseData.getApplicant1());
+
+            LanguagePreference languagePreference = caseData.getApplicant2().getLanguagePreference();
+
+            if (LanguagePreference.WELSH.equals(languagePreference)) {
```
Comment (mapingo) [inline src/main/java/uk/gov/hmcts/divorce/citizen/notification/Applicant2RequestChangesNotification.java:96]: Did you want to try the new CommonContent method here?

Comment (nitinprabhu) [reply src/main/java/uk/gov/hmcts/divorce/citizen/notification/Applicant2RequestChangesNotification.java:96]: yes will update all my PRs thanks

File: src/test/java/uk/gov/hmcts/divorce/citizen/notification/Applicant2RequestChangesNotificationTest.java [hunk 3]
```diff
@@ -215,6 +246,36 @@ void shouldSendEmailToApplicant2WithDissolutionContent() {
         verify(commonContent).mainTemplateVars(data, 1234567890123456L, data.getApplicant2(), data.getApplicant1());
     }
 
+    @Test
+    void shouldSendEmailToApplicant2WithDissolutionContentWhenLangPrefIsWelsh() {
+        CaseData data = caseData();
```
Comment (mapingo) [inline src/test/java/uk/gov/hmcts/divorce/citizen/notification/Applicant2RequestChangesNotificationTest.java:251]: Is this test needed now that the decision on setting Welsh content is inside CommonContent, and that is mocked here?

File: src/test/java/uk/gov/hmcts/divorce/citizen/notification/Applicant2RequestChangesNotificationTest.java [hunk 4]
```diff
@@ -192,6 +195,34 @@ void shouldSendEmailToApplicant2WithDivorceContent() {
         verify(commonContent).mainTemplateVars(data, 1234567890123456L, data.getApplicant2(), data.getApplicant1());
     }
 
+    @Test
+    void shouldSendEmailToApplicant2WithDivorceContentWhenLangPrefIsWelsh() {
```
Comment (mapingo) [inline src/test/java/uk/gov/hmcts/divorce/citizen/notification/Applicant2RequestChangesNotificationTest.java:199]: Is this test needed now that the decision on setting Welsh content is inside CommonContent, and that is mocked here?

**Pull request merged**